itests/src/test/resources/testconfiguration.properties

```diff
@@ -66,6 +66,7 @@ minillap.query.files=\
   external_table_purge.q,\
   external_table_with_space_in_location_path.q,\
   file_with_header_footer.q,\
+  float_failure.q,\
```

Comment:
This change is not necessary, q tests are only executed by `TestMiniLlapLocalCliDriver` by default.